Microsoft.Toolkit.Uwp.UI.Controls/Menu/MenuItem.cs

```diff
@@ -163,7 +166,12 @@ protected override void OnApplyTemplate()
             base.OnApplyTemplate();
         }
 
-        private void MenuItem_LayoutUpdated(object sender, object e)
+        private void MenuFlyout_Opening(object sender, object e)
+        {
```

Comment:
Nice catch :) I was testing something but it seems I removed the event registration but didn't remove the event it self